### PR TITLE
Refractor nav and small changes

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

File: fashify/src/components/Nav/Nav.jsx [hunk 1]
```diff
@@ -58,6 +58,9 @@ export const Nav = () => {
         </div>
         <div className='nav-right'>
           <ul className='nav-links'>
+            <li className='nav-link-item nav-explore-link'>
+              <Link to={'/products'}>Explore</Link>
+            </li>
```
Comment (poornima-alugubelly) [inline fashify/src/components/Nav/Nav.jsx:63]: even if `to` doesn't have braces and it's just a string then also it'll work  no need for braces

Comment (Rohanmond) [reply fashify/src/components/Nav/Nav.jsx:63]: I will change it Thanks

**Review by poornima-alugubelly — APPROVED** [review]
Clean code all good 👍

**Review by anuj-kumary — APPROVED** [review]
Everything looks good to me 
Good to merge

**Pull request merged**